docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -0,0 +1,288 @@
+# Provisioning hosted clusters on MCE (Without Hypershift Deployment)
+
+Configuring hosted control planes requires a hosting service cluster and a hosted cluster. By deploying the HyperShift operator on an existing cluster, you can make that cluster into a hosting service cluster and start the creation of the hosted cluster. 
+
+Hosted control planes is a Technology Preview feature, so the related components are disabled by default. Enable the feature by editing the `multiclusterengine` custom resource to set the `spec.overrides.components[?(@.name=='hypershift-preview')].enabled` to `true`. 
+
+Enter the following command to ensure that the hosted control planes feature is enabled, replacing `multiclusterengine-sample` with your MCE's instance name:
+
+```bash
+$ oc patch mce multiclusterengine-sample --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift-preview","enabled": true}]}}}'
```

Comment:
prefer to use the name `multiclusterengine` here but not the `multiclusterengine-sample`